.github/workflows/preview.yml

```
@@ -8,6 +8,8 @@ on:
       - "gulpfile.ts"
       - "package.json"
       - "tsconfig.json"
+    paths-ignore:
```

actionlint (1.7.7)

both "paths" and "paths-ignore" filters cannot be used for the same event "pull_request". note: use '!' to negate patterns

(events)